pyphare/pyphare/core/ufuncs.py

```
@@ -0,0 +1,168 @@
+    for time in h_.times():
+        interp_ = hier.interpol(time)
+        for lvl in h_.levels(time).values():
+            for patch in lvl.patches:
+                names = list(patch.patch_datas.keys())
+                box = patch.box
+
+                for name in names:
+                    pdata = patch.patch_datas[name]
+                    nb_ghosts = pdata.ghosts_nbr
+                    if not n_pad > nb_ghosts:
+                        raise ValueError('sigma value is too small')
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)